abc-bank/Accounts/SavingsAccount.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace abc_bank.Accounts {
+  public class SavingsAccount : AccountBase {
+
+    public override string ReportLabel { get; } = "Savings Account";
+
+    public override double InterestEarned {
```

Comment:
This is incorrect.  Instead of 'first 1000' it's 'if 1000 or less' apply .001% ...

Incidentally, were there a need for more complex or more case-based interest calcs, a State Pattern would likely be my preferred approach.  Overkill here though - and I even borked this bit up so...

Leaving it alone however - I was already approaching the 'perfect becoming the enemy of good' territory in other areas.